packages/ds-medicare-gov/src/styles/components/_override.choice.scss

```diff
@@ -1,58 +1,29 @@
-// positioning the radio
```

Comment:
Might be good to have an updated comment in here to give context for the override